node/src/main/resources/defaults.conf

```diff
@@ -317,6 +317,12 @@ casper {
     # Allow node being ceremony master
     # If this is `true` and no genesis block available on startup node will start genesis ceremony.
     ceremony-master-mode = ${standalone}
+
+    # Allow node starts from genesis block or Approved block
+    # If this is `true`,when a node without any data connecting to exist network, the node would starts from
+    # genesis block and validate blocks from scratch. Otherwise, the node would start from approved block which
+    # can save a lot of time of validating all the history blocks
+    from-last-finalized-state = true
```

Comment:
This should not be in `genesis-ceremony` section. I suggest to put it near `bootstrap`
And again - lets not use `last-finalized-state`.

 I suggest 

```
bootstrap = "botstrap_addr"
trim-state = true
```